javaSE/src/main/java/com/smartdevicelink/managers/lifecycle/LifecycleManager.java

```diff
@@ -72,7 +72,7 @@
 
     private static final String TAG = "Lifecycle Manager";
 
-    public static final Version MAX_SUPPORTED_RPC_VERSION = new Version("5.0.0");
+    public static final Version MAX_SUPPORTED_RPC_VERSION = new Version("5.1.0");
```

Comment:
As long as we're altering this line, please change to the other constructor `new Version(5, 1, 0);`